homeassistant/components/august/__init__.py

```diff
@@ -140,7 +140,7 @@ def __init__(
         self._config_entry = config_entry
         self._hass = hass
         self._august_gateway = august_gateway
-        self.activity_stream: ActivityStream | None = None
+        self.activity_stream: ActivityStream = None  # type: ignore[assignment]
```

Comment:
I went with a `type: ignore` here as `self.activity_stream` is actually accessed once before it's set.